sdk/servicebus/service-bus/test/internal/backupMessageSettlement.spec.ts

```diff
@@ -310,22 +329,30 @@ describe("Message settlement After Receiver is Closed - Through ManagementLink",
         "MessageId is different than expected"
       );
 
+      testLogger.info(`Attempting to complete the message: ${deadLetterMsgsBatch[0].messageId}`);
       await receiver.completeMessage(deadLetterMsgsBatch[0]);
-
       await testPeekMsgsLength(deadLetterReceiver, 0);
     } else {
       const messageBatch = await receiver.receiveMessages(1);
-      await receiver.completeMessage(messageBatch[0]);
 
+      testLogger.info(`Attempting to complete the message: ${messageBatch[0].messageId}`);
+      await receiver.completeMessage(messageBatch[0]);
       await testPeekMsgsLength(receiver, 0);
     }
+
+    testLogger.info(`Done testing dead letter`);
   }
 
-  it(
+  it.only(
     noSessionTestClientType + ": deadLetter() moves message to deadletter queue",
     async function(): Promise<void> {
-      await beforeEachTest(noSessionTestClientType);
-      await testDeadletter();
+      enableCommonLoggers();
```

Comment:
Are you going to remove the logging before merging? Or hide behind an env var?

Reply:
I'm leaving these on for the next little bit just so we can see more in case it fails. We can remove it later if we can get a couple of weeks without a failure.